SonarLint.VisualStudio.Integration.sln

```diff
@@ -90,9 +90,15 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IssueViz.Security.UnitTests
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Infrastructure.VS", "Infrastructure.VS", "{BE58232B-BE46-4D0D-9175-6BE0517F3EF0}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Infrastructure.VS", "src\Infrastructure.VS\Infrastructure.VS.csproj", "{55F623DC-7669-4D92-A04A-A963974F9438}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Infrastructure.VS", "src\Infrastructure.VS\Infrastructure.VS.csproj", "{9CBAB074-AB53-4AC5-85E0-FD064D71B170}"
```

Comment:
Something was messed up again with the project guids, but the changes are correct: 9CBAB074 is the Infrastructure's guid, and 407919AF is the test's guid.
The new typescript ones look correct, so hopefully it won't happen again next time we add another project.